docs/sources/administration/roles-and-permissions/access-control/_index.md

```diff
@@ -43,6 +43,8 @@ RBAC roles contain multiple permissions, each of which has an action and a scope
     - **Action:** `datasources:read`
     - **Scope:** `datasources:*`
 
+For information on the RBAC API refer to [RBAC API](https://grafana.com/docs/grafana/latest/developers/http_api/access_control/#rbac-api).
```

Comment:
We don't want all versions linking to "latest", each version of the docs should link to the same document in that version of docs. https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution does this

```suggestion
For information on the RBAC API refer to [RBAC API](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/developers/http_api/access_control/#rbac-api).
```